kvm/v12.0.0/README.md

```diff
@@ -0,0 +1,21 @@
+## :zap: Giant Swarm Release 12.0.0 for KVM :zap:
+
+This is the first Giant Swarm release which includes Kubernetes v1.17. Components have been updated for improved reliability and observability. Further details about changes to individual components can be found below.
+
+### Kubernetes v1.17.6
+- Updated from v1.16.9 - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v1175).
+- **Cloud Provider Labels reach General Availability**: Added as a beta feature way back in v1.2, v1.17 sees the general availability of cloud provider labels.
+- **Volume Snapshot Moves to Beta**: The Kubernetes Volume Snapshot feature is now beta in Kubernetes v1.17. It was introduced as alpha in Kubernetes v1.12, with a second alpha with breaking changes in Kubernetes v1.13.
+- **CSI Migration Beta**: The Kubernetes in-tree storage plugin to Container Storage Interface (CSI) migration infrastructure is now beta in Kubernetes v1.17. CSI migration was introduced as alpha in Kubernetes v1.14.
+
+### kvm-operator v3.11.2
+- Modified Calico deployment to use `-bird-live` as a liveness probe improving observability of failed mesh networking.
+- Updated node setup for 1.17 compatibility.
+- Removed limits from `calico-kube-controllers`.
```

Comment:
Can you explain why those limits were removed?

Reply:
We decided towards the end of last year to drop CPU and memory limits for all core components to ensure that they are not OOM killed. We replaced the limits with alerts about high memory usage which we can investigate and act on manually. Here's the issue for this specific change https://github.com/giantswarm/giantswarm/issues/10543

Reply:
```suggestion
- Removed limits from `calico-kube-controllers` to prevent it from being OOM killed.
```
How about this?